vspace/src/main/java/edu/asu/diging/vspace/web/ExhibitionModuleController.java

```diff
@@ -1,23 +1,48 @@
 package edu.asu.diging.vspace.web;
 
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
 
+import edu.asu.diging.vspace.core.model.IContentBlock;
+import edu.asu.diging.vspace.core.model.IModule;
+import edu.asu.diging.vspace.core.model.ISlide;
 import edu.asu.diging.vspace.core.services.IModuleManager;
+import edu.asu.diging.vspace.core.services.impl.ContentBlockManager;
 
 @Controller
 public class ExhibitionModuleController {
 
-    @Autowired
-    private IModuleManager moduleManager;
+	@Autowired
+	private IModuleManager moduleManager;
 
-    @RequestMapping(value = "/exhibit/module/{id}")
-    public String module(@PathVariable("id") String id, Model model) {
-        model.addAttribute("module", moduleManager.getModule(id));
+	@Autowired
+	private ContentBlockManager contentBlockManager;
 
-        return "module";
-    }
+	@RequestMapping(value = "/exhibit/module/{id}")
+	public String module(@PathVariable("id") String id, Model model, RedirectAttributes attributes) {
+		IModule module=moduleManager.getModule(id);
+		model.addAttribute("module", module);
+		try {
+			List<List<IContentBlock>> slideContents=new ArrayList<List<IContentBlock>>();
+			List<ISlide> slides = module.getStartSequence().getSlides();
+			Iterator<ISlide> iter=slides.iterator();
+			while(iter.hasNext()) {
+				ISlide slide=iter.next();
+				slideContents.add(contentBlockManager.getAllContentBlocks(slide.getId()));
+			}
+			model.addAttribute("slideContents", slideContents);
+		}
+		catch(Exception e) {
+			e.printStackTrace();
```

Comment:
nope, nope, nope, in so many ways. don't just catch all exceptions, catch specific ones. and why would an exception be thrown here? Handle that case appropriately. Show an error message or take otheer measures that will make sense from a user perspective. And this is not how you log an exception.